src/prefer-arrow-functions.ts

```diff
@@ -9,7 +9,7 @@ import {
 export default {
   meta: {
     docs: {
-      category: 'emcascript6',
+      category: 'ecmascript6',
```

Comment:
Unsure of the impact of this change

Reply:
I don't even see `category` as a field in the docs: https://eslint.org/docs/latest/extend/custom-rules#rule-structure so I'm not sure either. But I have to imagine the impact would be minimal